--- ./geos/include/geos/index/intervalrtree/IntervalRTreeBranchNode.h	2014-06-18 14:52:39.000000000 -0600
+++ ./geos/include/geos/index/intervalrtree/IntervalRTreeBranchNode_new.h	2014-06-18 14:59:05.000000000 -0600
@@ -47,7 +47,7 @@
 protected:
 public:
 	IntervalRTreeBranchNode( const IntervalRTreeNode * n1, const IntervalRTreeNode * n2)
-	:	IntervalRTreeNode( std::min( n1->getMin(), n2->getMin()), std::max( n1->getMax(), n2->getMax())),
+	:	IntervalRTreeNode( (std::min)( n1->getMin(), n2->getMin()), (std::max)( n1->getMax(), n2->getMax())),
 		node1( n1),
 		node2( n2)
 	{ }
